Dockerfile.dev

```diff
@@ -8,9 +8,9 @@ RUN apt-get update \
     dnsutils
 
 RUN cd /usr/local && \
-    wget http://apache.rediris.es/spark/spark-2.3.1/spark-2.3.1-bin-hadoop2.7.tgz && \
-    tar xzf spark-2.3.1-bin-hadoop2.7.tgz && \
-    ln -s spark-2.3.1-bin-hadoop2.7 spark
+    wget http://apache.rediris.es/spark/spark-2.3.2/spark-2.3.2-bin-hadoop2.7.tgz && \
```

Comment:
we'll have to update  again once 2.3.3 is released, we should find a mirror which doesn't delete old releases once new point releases are made.